upgrade.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+## Stop docker containers and pull images
+docker-compose down
+docker-compose pull
+
+## Start containers and run migrations
+docker-compose up -d redis seaquill
+
+docker-compose run --rm clortho /bin/clortho migrate up
+docker-compose run --rm warden migrate up
```

Comment:
Does this need to be `/bin/warden migrate up`?

Reply:
No. The Warden dockerfile has:
```
ENTRYPOINT ["/bin/warden"]
CMD ["serve"]
```

Which means that you don't have to specify the entrypoint again, but you can pass in a different cmd. Clortho just has

```
CMD ["/bin/wait-for-postgres.sh", "--", "/bin/clortho", "serve"]
```

So for it we have to be explicit since it doesn't have an entrypoint.